charts/tenant-base/chart/templates/networkpolicy.yaml

```
@@ -0,0 +1,54 @@
+{{- range .Values.deployments }}
+{{- if .networkPolicy }}
+{{- if or (gt (len .networkPolicy.ingress) 0) (gt (len .networkPolicy.egress) 0) }}
```

Base on [Helm ifelse](https://helm.sh/docs/chart_template_guide/control_structures/#ifelse) doc I think this could be simplifyed.

```suggestion
{{- if or .networkPolicy.ingress .networkPolicy.egress }}
```